front-end/src/components/Post/PostMotionInfo.tsx

```diff
@@ -0,0 +1,146 @@
+	const { depositAmount, metaDescription, method: preimageMethod, preimageArguments } = preimage || {};
+
+	return (
+		<div className={className}>
+			<h4>On-Chain Info</h4>
+			<Grid>
+				<Grid.Column mobile={16} tablet={8} computer={8}>
+					<div className='info_group'>
+						<h6>Proposer</h6>
+						{proposerAddress}
+					</div>
+				</Grid.Column>
+				<Grid.Column mobile={16} tablet={8} computer={8}>
+					<div className='info_group'>
+						<h6>Member Count</h6>
+						{memberCount}
+					</div>
+				</Grid.Column>
+				<Grid.Column mobile={16} tablet={16} computer={16}>
+					<div className='info_group'>
+						<h6>Motion hash</h6>
+						{motionProposalHash}
+					</div>
+				</Grid.Column>
+				<Grid.Row>
+					<Grid.Column mobile={16} tablet={8} computer={8}>
+						<div className='info_group'>
+							<h6>Motion&apos;s Method</h6>
+							{method}
+						</div>
+					</Grid.Column>
+					<Grid.Column mobile={16} tablet={8} computer={8}>
+						{motionProposalArguments && motionProposalArguments.length
+							? <div className='info_group'>
+								<h6>Arguments</h6>
+								{motionProposalArguments.map((element, index) => {
+									return <div className={'methodArguments'} key={index}>
+										<span key={index}>{element.name}: {element.value}</span>
+									</div>;
+								})}
+							</div>
+							: null}
+					</Grid.Column>
+				</Grid.Row>
+				<Grid.Row className='preimage'>
+					{depositAmount &&
+				<Grid.Column mobile={16} tablet={8} computer={8}>
+					<div className='info_group'>
+						<h6>Deposit</h6>
+						{parseInt(depositAmount) / Math.pow(10, chainProperties.kusama.tokenDecimals) + ' ' + chainProperties.kusama.tokenSymbol}
+					</div>
+				</Grid.Column>}
+					{preimageMethod &&
+				<Grid.Row>
+					<Grid.Column mobile={16} tablet={8} computer={8}>
+						<div className='info_group'>
+							<h6>Method</h6>
+							{preimageMethod}
+						</div>
+					</Grid.Column>
+					<Grid.Column mobile={16} tablet={8} computer={8}>
+						{preimageArguments && preimageArguments.length
+							? <div className='info_group'>
+								<h6>Arguments</h6>
+								{preimageArguments.map((element, index) => {
+									return <div className={'methodArguments'} key={index}>
+										<span key={index}>{element.name}: {element.value}</span>
+									</div>;
+								})}
+							</div>
+							: null}
+					</Grid.Column>
+				</Grid.Row>}
+					<Grid.Column mobile={16} tablet={16} computer={16}>
+						{ metaDescription &&
+					<div className='info_group'>
+						<h6>Description</h6>
+						{metaDescription}
+					</div>}
+					</Grid.Column>
+				</Grid.Row>
+			</Grid>
+		</div>
+	);
+};
+
+export default styled(PostMotionInfo)`
+	background-color: white;
+	padding: 2rem 3rem 2rem 3rem;
+	border-style: solid;
+	border-width: 5px;
+	border-color: grey_light;
+	font-size: sm;
+	overflow-wrap: break-word;
+	margin-bottom: 1rem;
+	font-family: 'Roboto Mono';
+
+	h6 {
+		font-family: 'Roboto Mono';
```

Comment:
is font-family needs to be defined twice. it is already defined on 121

Reply:
It's needed because it overriding a style set globally on `h6`.